tests/test_github_stargazers.py

```diff
@@ -150,7 +148,8 @@ def test_stargazer_on_invalid_page(url_page_content: str,
     )
     runner = CliRunner()
     result = runner.invoke(command_line, ['foo/bar', '--user', 'foo'])
-    verify_invoke_from_clirunner(result, http_not_found)
+    message = http_not_found("foo/bar")
+    verify_invoke_from_clirunner(result, message)
```

Comment:
One-liner here, since `message` is used only in one place. Same for the same case in other tests.